src/components/interior-left-nav/_interior-left-nav.scss

```diff
@@ -262,7 +262,7 @@
 
       &:hover,
       &:focus {
-        background-color: rgba(#3d70b2, 0.3);
+        background-color: rgba($link-01, 0.3);
```

Comment:
- [ ] Change to hex and override with `body.bx--v9-theme`  or find a token due to the css custom properties issue or other solution.